.travis.yml

```diff
@@ -4,13 +4,12 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
```

Comment:
The only change I can think of that is breaking `OS X` is this one. The failure is:

```shell
ld: library not found for -lgcc_s.10.5
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [qtdirs.app/Contents/MacOS/qtdirs] Error 1
Error: Failed to determine the layout of your Qt installation. Try again using
the --verbose flag to see more detail about the problem.
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/cli/main_build.py", line 321, in main
    execute(sys.argv[1:])
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/cli/main_build.py", line 312, in execute
    noverify=args.no_verify)
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/api.py", line 89, in build
    need_source_download=need_source_download, config=config)
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/build.py", line 1460, in build_tree
    config=config)
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/build.py", line 1043, in build
    utils.check_call_env(cmd, env=env, cwd=src_dir)
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/utils.py", line 590, in check_call_env
    return _func_defaulting_env_to_os_environ(subprocess.check_call, *popenargs, **kwargs)
  File "/Users/travis/miniconda3/lib/python3.5/site-packages/conda_build/utils.py", line 586, in _func_defaulting_env_to_os_environ
    return func(_args, **kwargs)
  File "/Users/travis/miniconda3/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/bash', '-x', '-e', '/Users/travis/miniconda3/conda-bld/recipe_1484890824608/work/PyQt-mac-gpl-4.11.4/conda_build.sh']' returned non-zero exit status 1
```